Documentation totp device (#1545)
kartoza/sawps: docs/src/administrator/manual/django-table.md

> ## Activities Table
> 
> ![Django Admin Table](./img/django-table-1.png)
> 
> 1. **Add button**: Clicking on this button will allow user to add a new record in the table. Clicking on [add](django-add-data.md) will navigate you to add data record documentation.
> 
> 2. **Search**: Search feature to search specific record of the table.
> 
> 3. **Toggle select records**: Checkbox to select/deselect records.
> 
> 4. **Action**: Users can choose the action from this dropdown and perform it on a selected object.
> 
> 5. **Go**: Clicking on this button will allow users to perform the selected option on the selected object.
> 
> 6. **Objects**: Available objects containing records. Users can see the details of the object by clicking on the link.
> 
> ## Group Table
> 
> ![Group Table](./img/django-table-2.png)
> 
> This table is used to store existing user groups, that will affect available options in the [Report](../../user/manual/explore/reports.md).
> Administrators can assign a user to groups in the User Table.
> 
> 1. **Toggle sorting**: Clicking on this icon will allow users to toggle data.
> 
> 2. **Add Group**: Clicking on the `ADD GROUP` button will allow users to add a new group. Click on [add group](django-add-data.md) to see detailed documentation about adding a new group.
> 
> 3. **Edit Record**: Clicking on the object will allow users to change/edit the particular record. Click on [change group](django-change-data.md) to see detailed documentation about editing a group.

Reviewer: Another reversion of a previously correct file?
```suggestion
# Django Tables

## Activities Table

![Django Admin Table](./img/django-table-1.png)

1. **Add button**: Clicking on this button will allow administrators to add a new record in the table. Clicking [here](django-add-data.md) will redirect users to the add data record documentation.

2. **Search**: Feature to allow for searching of a specific object in the table using keywords.

3. **Toggle select records**: Checkbox to select/deselect records.

4. **Action**: Administrators can choose the action from this dropdown and then perform it on a selected object.

5. **Go**: Clicking on this button will allow administrators to perform the selected option on the selected object.

6. **Objects**: Available objects containing records. Administrators can see the details of the object by clicking on the link (i.e. the object name).

## Group Table

![Group Table](./img/django-table-2.png)

This table is used to store existing user groups, that will affect the available options in the [Report](../../user/manual/explore/reports.md).

Administrators can assign a user to groups in the User Table.

1. **Toggle sorting**: Clicking on this icon will allow administrators to toggle the displayed order of the data.

2. **Add Group**: Clicking on the `ADD GROUP` button will allow administrators to add a new group. Click [here](django-add-data.md) to see detailed documentation about adding a new group.

3. **Edit Record**: Clicking on the object will allow administrators to change/edit a particular record. Click [here](django-change-data.md) to see detailed documentation about editing a group.
```

Author: This will solve after the rebase

## TOTP(Time-based One-Time Passwords) Device

![Totp Device](./img/django-table-6.png)

This table is used for storing, generating, deleting the TOTP device.

1.  **TOTP Table**: The table displays record for the TOTP device.

2.  **QRCode**: The administrator can view the QRCode of the TOTP device by clicking on the `qrcode` link.

    - ![QRCode](./img/django-table-7.png)
    1. **QRCode**: Displays QRCode.
    2. **URI**: URI(Uniform Resource Identifier) for configure TOTP.

> 3. **Add TOTP Device**: The administrator can add TOTP device by clicking on the `ADD TOTP DEVICE` button. Click on the [add TOTP device](./django-add-data.md) to view detailed documentation about adding a new TOTP device.

Reviewer: ```suggestion
3. **Add TOTP Device**: The administrator can add TOTP device by clicking on the `ADD TOTP DEVICE` button. Click [[here](./django-add-data.md) to view detailed documentation about adding a new TOTP device.
```

> 4. **Edit TOTP Device**: The administrator can edit the TOTP device for that particular object by clicking on the object. Click on the [edit TOTP device](./django-change-data.md) to view detailed documentation about editing a TOTP device.

Reviewer: ```suggestion
4. **Edit TOTP Device**: The administrator can edit the TOTP device for that particular object by clicking on the object. Click on the [here](./django-change-data.md) to view detailed documentation about editing a TOTP device.
```

Reviewer: This hasn't been applied

4. **User Table**: The user table with records.

5. **Edit User**: Clicking on the object allows the administrators to change or edit a particular record. Click [here](django-change-data.md) to view detailed documentation on editing a user.

> ## TOTP(Time-based One-Time Passwords) Device

Reviewer: ```suggestion
## TOTP (Time-based One-Time Passwords) Device
```
